dcm2bids/helper.py

```diff
@@ -6,38 +6,39 @@
 import os
 import sys
 from .dcm2niix import Dcm2niix
-from .utils import DEFAULT
-
-
-def get_arguments():
-    """Load arguments for main"""
-    parser = argparse.ArgumentParser(
-        formatter_class=argparse.RawDescriptionHelpFormatter,
-        description="",
-        epilog="""
-            Documentation at https://github.com/unfmontreal/Dcm2Bids
-            """,
-    )
-
-    parser.add_argument(
-        "-d", "--dicom_dir", required=True, nargs="+", help="DICOM files directory"
-    )
-
-    parser.add_argument(
-        "-o",
-        "--output_dir",
-        required=False,
-        default=DEFAULT.cliOutputDir,
-        help="Output BIDS directory, Default: current directory",
-    )
+from .utils import DEFAULT, assert_dirs_empty
 
-    args = parser.parse_args()
-    return args
+EPILOG = """
+    Documentation at https://github.com/unfmontreal/Dcm2Bids
+    """
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument("-d", "--dicom_dir",
+                   required=True, nargs="+",
+                   help="DICOM files directory.")
+
+    p.add_argument("-o", "--output_dir",
+                   required=False, default=DEFAULT.cliOutputDir,
+                   help="Output BIDS directory."
+                        " (Default: %(default)s)")
+
+    p.add_argument('-f',
```

Comment:
```suggestion
    p.add_argument('--force',
```
Better be safe than sorry, `